src/jsdoc.ts

```diff
@@ -253,7 +264,17 @@ function tagToString(tag: Tag, escapeExtraTags = new Set<string>()): string {
 const SINGLETON_TAGS = new Set(['deprecated']);
 
 /** Serializes a Comment out to a string usable in source code. */
+export function getContents(tags: Tag[], escapeExtraTags = new Set<string>()): string {
+  return serialize(tags, false, escapeExtraTags);
+}
+
+/** Serializes a Comment out to a string usable in source code. */
```

Comment:
This comment is identical to the one above, but it seems the functions do different things?

Reply:
Done, and renamed the function.